### ISSUE TITLE
Crash on iOS createBanner

### ISSUE THREAD

**Issue by seibelj** [floatinghotpot/cordova-admob-pro#335]
Getting EXC_BAD_ACCESS on this line of CADVAdMobPlugin.m line 131 with latest release when running on iOS. I'm using the simulator.

`if(* [adId UTF8String] == '/') {`

It happens when I call `AdMob.showBanner(admobid.position)`; in my JS code. This code was originally written about 6 months ago. When I remove that line, the banner autoshows and there is no crash. I wonder why this stopped working?

**Comment by floatinghotpot** [comment]
I will look into this issue. what's your version of plugin, Xcode and iOS?

**Comment by seibelj** [comment]
Plugin: 2.13.0 
XCode: 7.2
Simulator: 9.2

**Comment by floatinghotpot** [comment]
Did you set AdId first? It seems an empty adId will trigger the bug. 

I will add checking the adId for safety, but a valid logic should be creating banner with a given adId, then call showBanner() later.

**Comment by seibelj** [comment]
The code is like this:

```
AdMob.createBanner( {
                            adId: admobid.banner, 
                            isTesting: admobid.is_testing,
                            overlap: false, 
                            offsetTopBar: false, 
                            position: position,
                            bgColor: 'black',
                        } );
AdMob.showBanner(admobid.position);
```

`admobid` is an object containing:

```
admobid = {
        banner: 'ca-app-pub-692...', // or DFP format "/6253334/dfp_example_ad"
        interstitial: 'ca-app-pub-6920902...',
        type: 'iOS',
        is_testing: true
    };
```

**Comment by floatinghotpot** [comment]
In fact, you don't have to call showBanner() like this, just add one more param when createBanner(): 
`autoShow:true`

**Comment by seibelj** [comment]
Right, I notice that now, but it worked fine before. I changed my code, it was just odd that it was a race condition, I feel like it shouldn't be possible and worries me about random crashes.

**Comment by floatinghotpot** [comment]
Fixed, safe check added in 8e69bcd4f2532d2b7c7354fdcdeda6f86d36813b
